projects/packages/forms/src/blocks/contact-form/util/form-styles.js

```diff
@@ -143,6 +145,7 @@ window.jetpackForms.generateStyleVariables = function ( formNode ) {
 		'--jetpack--contact-form--border': border,
 		'--jetpack--contact-form--border-color': borderColor,
 		'--jetpack--contact-form--border-size': borderWidth,
+		'--jetpack--contact-form--border-size-outline': borderWidthOutline,
```

Comment:
If this is meant to be a fallback for Outlined style I'd say let's use the same wording "outlined" (specially since `outline` is a different CSS property). Easy fix though.